CHANGELOG.md

```diff
@@ -25,8 +25,10 @@ The following is a curated list of changes in the Enact agate module, newest cha
 - `agate/IncrementSlider` button color for Gallium skin
 - `agate/Input` run time error when using `dismissOnEnter`
 - `agate/Keypad` to not show console error in sampler
+- `agate/Keypad` to have buttons with `minWidth` false
```

Comment:
I think that this CHANGELOG seems to be too detail to app developers.
App developers doesn't interest in the internal code.
So it might be better to fix it like the following way.
```
- `agate/Keypad` to have buttons with proper width
```

Reply:
ok, I updated it